src/main/java/systems/composable/dropwizard/cassandra/connect/IdentityTranslatorFactory.java

```diff
@@ -0,0 +1,14 @@
+package systems.composable.dropwizard.cassandra.connect;
+
+import com.datastax.driver.core.policies.AddressTranslator;
+import com.datastax.driver.core.policies.IdentityTranslator;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+/** A factory for configuring and building {@link com.datastax.driver.core.policies.IdentityTranslator} instances. */
+@JsonTypeName("identityTranslator")
```

Comment:
What do we gain by defining this type? It looks like its equivalent to not specifying an `addressTranslator`.